Error after following instructions in README

Newly cloned repo, following the instructions in the README file, I get:

```
...
Creating casesvc                ... done
Creating iac                    ... done
Creating sample                 ... done
Creating survey                 ... done
Creating actionexporter         ... done
Creating party-service          ... done
Creating cfdatabasetool         ... done
Creating frontstage             ... done
Creating comms-template         ... done
Creating oauth2-service         ... done
Creating secure-message         ... done
Creating redis                  ... done
Creating collection-instrument  ... done
Creating rabbitmq               ... done
Creating response-operations-ui ... done
Creating notifygateway          ... done
Creating postgres               ... done
Creating respondent-home-ui     ... done
Creating sftp                   ... done
Creating sdx-gateway            ... done
Creating action                 ... done
Creating collex                 ... done
Creating uaa                    ... done
Creating pgadmin                ... done
oauth2-service
oauth2-service
pipenv install --dev
make: pipenv: No such file or directory
make: *** [up] Error 1
```

--- a/README.md
+++ b/README.md
@@ -7,6 +7,17 @@ A combined repository for unifying approach to running RAS and RM in Docker
 1. Run `docker login` in a terminal and use your docker hub account
 1. Run `docker network create rasrmdockerdev_default` to create the docker network
 
+## Setup
+Based on python 3.6
+
+Use [Pyenv](https://github.com/pyenv/pyenv) to manage installed Python versions
+
+[Pipenv](https://docs.pipenv.org/) is required locally for running setup scripts
+
+```bash
+pip install -U pipenv
+```
+
 ## Quickstart
 ![make up](https://media.giphy.com/media/xULW8lyhMJjzyO33sA/giphy.gif)
 ```

